packages/esm-patient-tests-app/src/test-orders/add-test-order/add-test-order.workspace.tsx

```diff
@@ -8,22 +8,22 @@ import {
   ArrowLeftIcon,
   getPatientName,
   formatDate,
+  launchWorkspace,
   parseDate,
   useLayoutType,
   useConfig,
 } from '@openmrs/esm-framework';
 import {
   type DefaultPatientWorkspaceProps,
   type OrderBasketItem,
-  launchPatientWorkspace,
   useOrderType,
   usePatientChartStore,
 } from '@openmrs/esm-patient-common-lib';
+import { type ConfigObject } from '../../config-schema';
+import type { TestOrderBasketItem } from '../../types';
```

Comment:
Nit: shouldn't these imports be at the last just before the styles, since they are types?

Reply:
I'm not entirely certain about the import order regarding type imports. I think we should debate this when we enforce [sorted imports](https://eslint.org/docs/latest/rules/sort-imports) in our ESLint config. For now, I'm more concerned with ensuring the main canonical groupings are observed. That looks something like this to me:

- React and core dependencies first
- Third-party libraries (`classnames`, `lodash-es`, `react-i18next`, `@carbon/react`)
- Internal ilibrary imports (`@openmrs/*`)
- Local imports (relative paths - more open to nuance)